File: A. Boy or Girl/python/aboyorgirl.py
x = input()
a = []
result = 0
for letter in x :
    if (letter not in a):
        result = result + 1 
    a.append(letter) 
print(a)
if (result % 2 == 1):
    print("IGNORE HIM!")
else:
    print("CHAT WITH HER!")

File: A. Boy or Girl/dart/aboyorgirl.dart
import 'dart:io';

main() {
  String x, leter;
  List a, s;
  int result, i;
  x = stdin.readLineSync();
  s = [];
  a = [];
  result = 0;
  a = x.split(RegExp(""));
  s = a.toSet().toList();
  result = s.length;
  if (result % 2 == 1) {
    print("IGNORE HIM!");
  } else {
    print("CHAT WITH HER!");
  }
}
